Altis_Life.Altis/dialog/MasterHandler.hpp

```diff
@@ -27,4 +27,6 @@
 #include "federalReserve.hpp"
 #include "compensate.hpp"
 #include "FuelStat.hpp"
-#include "newsStation.hpp"
\ No newline at end of file
+#include "AltisPhone.hpp"
```

Comment:
This was removed @dbenson24  .. make sure you have the most updated files, cause it doesn't look like it :<